### DISCUSSION TITLE
Ulauncher closes when Ctrl is pressed | Pasting from clibpoard not possible

### DISCUSSION THREAD

**Discussion by SHU-red** [Ulauncher/Ulauncher#906] [General]
Did search for similar stuff before.
Did not find a matching one ...

### Steps to reproduce
1. Press Alt+Space (my Hotkey to open Ulauncher) --> Ulauncher opens
2. Press Ctrl (no matter if lCtrl or rCtrl), as preparation to hit Ctrl+V for inserting strings from clipboard

### Current behavior
Ulauncher disappears

### Expected behavior
Ulauncher stays active
I can hit Ctrl-V to insert stuff from Clipboard

### EDIT AFTERWARDS
I tried one thing --> I have activated the "Mouse and Touchpad"-system-option for highlighting the mouse-cursor if Ctrl is pressed
It seems that this is causing the error
As soon as i deactivate this option (and cursor is not highlighted andymore) Ulauncher stays open and i can paste stuff

### Environment
**Ulauncher version**: v5.9v0~beta1
**Linux distro**: Linux Mint 20 Ulyana / Ubuntu 20.04 focal
**Desktop environment**: Cinnamon 4.6.7

### Logs
https://gist.github.com/SHU-red/1d3b4b034580a7ace327716f0ded9075

**Comment by troycurtisjr** [comment]
I suspect this is because the focus is being stolen away from ulauncher when the mouse-cursor is being highlighted. This is a long shot but can you try enabling the "Don't hide after losing mouse focus" option in the ULauncher preferences. There is a chance it might work, though I suspect the change is likely due to *keyboard* focus and not mouse focus.

**Comment by kwon37xi** [comment]
I had the same problem on Ubuntu / Gnome.
"Don't hide after losing mouse focus" does not work.
I found this was because of Gnome tweak-tool's Show pointer location with Ctrl key" feature.
After turning off the feature, Ctrl-V works.

But anyway I think "Don't hide after losing mouse focus" should work even when "Show pointer location" is on.
